test/screenshot/spec/mdc-menu/fixture.scss

```diff
@@ -30,3 +30,8 @@
 .menu-button-container {
   position: relative;
 }
+
+.menu-baseline {
```

Comment:
Rename all CSS classes to `test-menu-*` so it's clear that they're screenshot-specific